autoload/lsp/ui/vim.vim

```diff
@@ -606,16 +606,23 @@ endfunction
 "   server - string
 "   comand_or_code_action - Command | CodeAction
 function! s:execute_command_or_code_action(server, command_or_code_action) abort
-    if has_key(a:command_or_code_action, 'command') && type(a:command_or_code_action['command']) == type('')
-        let l:command = a:command_or_code_action
-        call s:execute_command(a:server, l:command)
+    let l:code_action = a:command_or_code_action
+
+    if has_key(l:code_action, 'arguments') && type(l:code_action['arguments']) == type([])
+        for l:action in l:code_action['arguments']
```

Comment:
My understanding is that `arguments` can be any data, and is only used at the server side. I don't think you should be interpreting this at the client?